recipes/essm/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "essm" %}
+{% set version = "0.4.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7a66cd8ebb52b92ba8880a13441dceb23b1e88c0f4c7e08e3a806eef65985d16
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.0
+    - pip
+    - setuptools-scm >=3.1.0
+  run:
+    - python >=3.0
+    - sympy ==1.3
```

Comment:
Does this need to be so tight of a pin?

Reply:
Unfortunately yes, for the current version. There is a [PR](https://github.com/environmentalscience/essm/pull/90) waiting in essm to relax the sympy pinning, which will become the next version. Once we include one version in conda-forge, will it update automatically, or will I have to re-do the recipe for each update where pinning changes?

Reply:
It uses features introduced in or just before sympy 1.3. I will update the PR to use the most recent version of sympy, which should work with sympy>=1.6.0

Reply:
So it should be <=1.3 then?

Reply:
Thanks for looking into this. I just updated to the new version of essm with clearer dependencies. Hope this solves the issue about the sympy version, but the pytest-runner problem will probably prevail.